mitodl/micromasters
# Added medium sized image to profile model and REST API

#### What are the relevant tickets?
Fixes #2213

#### What's this PR do?
Makes `image_medium` for the `Profile` model and the `/api/v0/profiles/.../` REST API. 

#### How should this be manually tested?
Run the migrations then verify that you see a new `image_medium` URL in `/api/v0/profiles/<username_here>/` and that it is 128x128 pixels. 

#### Any background context you want to provide?
@roberthouse54 What pixel dimensions should this image have?
